index.js

```diff
@@ -1,6 +1,19 @@
+const express = require("express")
+const app = express()
+const departments = require("./departments")
 const scrape = require("./scrape")
+const bookshelf = require("./bookshelf")
 
-const departments = require("./departments")
+app.set("bookshelf", bookshelf)
+
+app.use((req, res, next) => {
+  res.header("Access-Control-Allow-Origin", "*")
+  next()
+})
+
+app.listen(3000, () => {
+  console.log("API Listening on port 3000")
+})
```

Comment:
This block for setting the `Access-Control-Allow-Origin` header and listening on the port may be premature for this PR.

I'll probably remove this before merging.